test_tf2/test/test_convert.cpp

```diff
@@ -96,7 +97,91 @@ TEST(tf2Convert, kdlBulletROSConversions)
   EXPECT_NEAR(b1.z(), b4.z(), epsilon);
 }
 
-int main(int argc, char** argv)
+TEST(tf2Convert, ConvertTf2Quaternion)
+{
+  const tf2::Quaternion tq(1, 2, 3, 4);
+  Eigen::Quaterniond eq;
```

Comment:
Add needed includes for these at the top:

```
#include <tf2/LinearMath/Quaternion.h>
#include <Eigen/Eigen>
```

Also, I think we need to declare a dependency on `eigen` in the package.xml and the CMakeLists.txt.